src/dotnet/commands/dotnet-tool/restore/ToolRestoreCommand.cs

```diff
@@ -163,6 +179,9 @@ public override int Execute()
                 return 1;
             }
 
+            _reporter.WriteLine("Restore was successful.");
```

Comment:
Should we perhaps say Local tools restore was successful? Also, shouldn't this be localized?

Reply:
fixed